companion/src/firmwares/generalsettings.h

```diff
@@ -231,7 +231,7 @@ class GeneralSettings {
     bool noJitterFilter;
     bool rtcCheckDisable;
     bool keysBacklight;
-    bool rotEncDirection;
+    unsigned int rotEncDirection;
```

Comment:
Please rename to "rotEncMode", or, since the last word gets shorter, "rotEncoderMode"